Modules/Scripted/SegmentStatistics/SegmentStatisticsCalculators/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(${MODULE_NAME}Calculators_PYTHON_SCRIPTS
+  #${MODULE_NAME}Calculators
+  __init__
+  ${MODULE_NAME}CalculatorBase
+  Labelmap${MODULE_NAME}Calculator
```

Comment:
Hi @lassoan! Thank you for your constructive review. I made the requested changed. A point by point response to your comments follows below.

> use SegmentStatistics instead of ${MODULE_NAME

done

> replace "Calculator" by "Plugin"

done